src/main/java/dinkplugin/domain/ExceptionalDeath.java

```diff
@@ -10,6 +10,7 @@ public enum ExceptionalDeath {
     COX("Chambers of Xeric"),
     FIGHT_CAVE("Fight Caves"),
     INFERNO("Inferno"),
+    JAD_CHALLENGES("Jad challenges"),
```

Comment:
```suggestion
    JAD_CHALLENGES("TzHaar-Ket-Rak Challenges"),
```
I'm fine keeping the enum name, but the user-facing one needs to be recognizable, Jad doesn't offer these challenges

Reply:
```suggestion
    JAD_CHALLENGES("Inferno Challenges"),
```
Discussed in chat, full nametoo long for the section of the settings. only other idea which seems more-correct

Reply:
As discussed in chat, keeping as Jad challenges - can be changed later if people are super confused